src/utility_function_parameters.jl

```diff
@@ -0,0 +1,274 @@
+"""
+  Compute utility function parameters by region considered in the model (uk, eu, rest of world).
+  Parameters of this type appear in multiple utility functions in the paper, and are annotated
+  (at least) α, β1, β2, γ and ρ.
+
+  This is refactored from the Matlab code in e.g. ComputeTheta lines 62-64
+
+  # Arguments
+   - elasticity [ϵ, χ1, χ2, η, ξ]
+   - log_price_{uk, eu, w} [log(p)]
+   - quantity_{uk, eu, w} [f, x1, x2, I, m]
+
+  # Outputs
+  - weight_{uk, eu, w}
+
+"""
+function parameters_by_region(elasticity::T,
+                           log_price_uk::T,log_price_eu::T,log_price_world::T,
+                           quantity_uk::T,quantity_eu::T,quantity_world::T) where {T <: Real}
+
+    logP = log_price_index(elasticity, log_price_uk, log_price_eu, log_price_world, quantity_uk, quantity_eu, quantity_world)
+
+    weight_uk = quantity_uk * exp((elasticity - 1.0) * (log_price_uk - logP))
+    weight_eu = quantity_eu * exp((elasticity - 1.0) * (log_price_eu - logP))
+    weight_world = quantity_world * exp((elasticity - 1.0) * (log_price_world - logP))
```

Comment:
```suggestion
    weight_uk = quantity_uk * exp((elasticity - 1) * (log_price_uk - logP))
    weight_eu = quantity_eu * exp((elasticity - 1) * (log_price_eu - logP))
    weight_world = quantity_world * exp((elasticity - 1) * (log_price_world - logP))
```
I feel like it'd be better to have a function for this (so that the suggested change would be on a single line, instead of 3 :upside_down_face:)